packages/venia-ui/lib/components/CheckoutPage/checkoutPage.gql.js

```diff
@@ -0,0 +1,42 @@
+import gql from 'graphql-tag';
+import { CheckoutPageFragment } from './checkoutPageFragments.gql';
+
+const CREATE_CART_MUTATION = gql`
```

Comment:
I think these operations should also be exported individually. Your suggestion to also roll them up into a single default export for convenience is something I'm willing to run with, though.

```suggestion
export const CREATE_CART_MUTATION = gql`
```